softwareComponents/configuration/include/configuration/universalModule.hpp

```diff
@@ -34,18 +40,18 @@ class UniversalModule : public Module {
         return _getJointAngle( 2 );
     }
 
-    Angle setAlpha( Angle a ) {
-        setJointParams( 0, a.rad() );
-        return getAlpha();
+    void setAlpha( Angle a ) {
+        std::vector< float > tmp{ a.rad() };
```

Comment:
You can create a `span` from an `array` - no need to allocate a one element vector.